src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -0,0 +1,597 @@
+                }
+                validator = ValidateAIF.create(ImmutableSet.copyOf(domainOntologySources), restriction);
+                builder.setLength(builder.length() - 1);
+                ontologyStr = builder.toString();
+            }
+        } catch (RuntimeException rte) {
+            logger.error("Could not read/parse all domain ontologies or SHACL files...exiting.");
+            logger.error("--> " + rte.getLocalizedMessage());
+            return ReturnCode.FILE_ERROR.ordinal();
+        }
+
+        validator.setThreadCount(threads);
+        boolean hasFiles = files != null;
+
+        // Collect the file(s) to be validated.
+        final List<File> filesToValidate = new ArrayList<>();
+        int nonTTLcount = 0;
+        if (hasFiles) {
+            for (File file : files) {
+                if (file.getName().endsWith(".ttl")) {
+                    filesToValidate.add(file);
+                } else {
+                    logger.warn("Skipping file without .ttl suffix: " + file);
+                    nonTTLcount++;
+                }
+            }
+        } else { // -d option
+            File dir = directory;
+            if (!dir.exists()) {
+                logger.warn("Skipping non-existent directory: " + dir.getName());
+            } else if (dir.isDirectory()) {
+                File[] files = dir.listFiles(pathname -> pathname.toString().endsWith(".ttl"));
+                if (files != null) {
+                    filesToValidate.addAll(Arrays.asList(files));
+                }
+            } else {
+                logger.warn("Skipping non-directory: " + dir.getName());
+            }
+        }
+
+        if (filesToValidate.isEmpty()) {
+            logger.error("No files with .ttl suffix were specified.  Use -h option for help.");
+            return ReturnCode.FILE_ERROR.ordinal();
+        }
+
+        // Display a summary of what we're going to do.
+        if (hasFiles) {
+            logger.info("-> Validating KB(s): " +
+                    (filesToValidate.size() <= 5 ? filesToValidate : "from command-line arguments."));
+        } else { // We'd have failed by now if there were no TTL files in the directory
+            // This would need to be addressed if we supported validating files in N directories.
+            logger.info("-> Validating all KBs (*.ttl) in directory: " + directory.getName());
+        }
+        logger.info("-> Validating with domain ontology(ies): " + ontologyStr);
+        if (restriction == ValidateAIF.Restriction.NIST) {
+            logger.info("-> Validating against NIST SHACL.");
+        } else if (restriction == ValidateAIF.Restriction.NIST_TA3) {
+            logger.info("-> Validating against NIST Hypothesis SHACL.");
+        }
+        if (abortSet) {
+            logger.info("-> Validation will abort after " + maxValidationErrors + " SHACL violation(s).");
+            validator.setAbortThreshold(maxValidationErrors);
+        }
```

Comment:
Like other features, number of threads should be logged in the "Display a summary of what we're going to do" section.  Something like:

```suggestion
        }
        if (threads != MINIMUM_THREAD_COUNT) {
            logger.info("-> Validation will occur in " + threads + " threads.");
            validator.setThreadCount(threads);
        }
```

Reply:
Fixed